.editorconfig

```diff
@@ -0,0 +1,16 @@
+# https://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+indent_style = tab
+tab_width = 4
```

Comment:
Actually I use 8-space tabs, because I consider it the standard for C code.

I think 4 or 8 shouldn't matter with cautious use of tabs (only at start of line and only for adding nesting levels), but I can't rule out that some spacing might get mangled.

I bring this up because adding this to the `.editorconfig` makes GitHub display the code with 4-space tabs.

Reply:
Sorry, fixed.

Reply:
Given `indent_style = tab`, can't `tab_width = 4` just be omitted as it's just cosmetic & will not modify in anyway what will end up in the files ?
I'd rather use user default setting for this one (there's also a default per user GitHub setting for this).